### PR TITLE
some minor fix in test/e2e/kubectl/kubectl.go and remove some unused functions

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
**What this PR does / why we need it**:
some minor fix in test/e2e/kubectl/kubectl.go
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/kubectl/kubectl.go [hunk 1]
```diff
@@ -255,7 +252,7 @@ var _ = SIGDescribe("Kubectl client", func() {
 		if err != nil || len(pods) < atLeast {
 			// TODO: Generalize integrating debug info into these tests so we always get debug info when we need it
 			framework.DumpAllNamespaceInfo(f.ClientSet, ns)
-			framework.Failf("Verified %v of %v pods , error : %v", len(pods), atLeast, err)
+			framework.Failf("Verified %v of %v pods , error: %v", len(pods), atLeast, err)
```
Comment (neolit123) [inline test/e2e/kubectl/kubectl.go]: given these are integers:
`Verified %d of %d pods, error: %v", len(pods), atLeast, err)`
also fixes space before comma.

File: test/e2e/kubectl/kubectl.go [hunk 2]
```diff
@@ -1993,21 +1994,6 @@ type updateDemoData struct {
 
 const applyTestLabel = "kubectl.kubernetes.io/apply-test"
 
-func readBytesFromFile(filename string) []byte {
```
Comment (neolit123) [inline test/e2e/kubectl/kubectl.go]: it's always nice to add a sentence or two in the PR description about removing unused code.

Comment (hanxiaoshuai) [reply test/e2e/kubectl/kubectl.go]: Done

File: test/e2e/kubectl/kubectl.go (hunk 1 above)
Comment (hanxiaoshuai) [reply test/e2e/kubectl/kubectl.go]: Done, and Thanks for review

**Pull request merged**